### DISCUSSION TITLE
Implementing GAS labels

### DISCUSSION THREAD

**Discussion by MartsINY1** [cc65/cc65#1866] [Ideas]
Those labels would be really useful if implemented.

**Comment by polluks** [comment]
Why GNU assembler?

**Comment by Movax12** [comment]
Made this for fun. See comments posted in the file for usage, limitations: https://gist.github.com/Movax12/b11687f218f38952ecf2dcb98c1756b0

**Reply by MartsINY1** [reply]
thanks, from what I checked it seem fine
Is it in the main branch?
do you think you'd be good to check that one too?
https://github.com/cc65/cc65/discussions/1864
going to switch my whole engine from asm6 once this is considered

**Reply by Movax12** [reply]
The file I posted is an implementation of GNU assembler numbered local labels using CA65 macros. To use the macros, include the file in your build. There are limitations due to what I could get the macro code to do, but it works. I am not planning to add a PR in C for this. I'm not sure if https://github.com/cc65/cc65/discussions/1864 is possible or what it would look like with macros.

**Reply by MartsINY1** [reply]
can you explain how it may not be possible?

**Reply by MartsINY1** [reply]
> Made this for fun. See comments posted in the file for usage, limitations: https://gist.github.com/Movax12/b11687f218f38952ecf2dcb98c1756b0

Could I ask you to take a look at this one?

https://github.com/cc65/cc65/discussions/1864

I could try to switch my project if this is available
